### DISCUSSION TITLE
Questions to customer on 21st of October

### DISCUSSION THREAD

**Discussion by altanborali16** [altanborali16/SWE574-Group2#23] [General]
Please add your questions... @efestrikesback @AliSerGok @EnesAkkal  @kotobusdriver @Stroggification

**Comment by kotobusdriver** [comment]
Of the new features requested (gamification, aka badges, and recommendation systems), which one is of higher priority (and should be implemented first)?

**Comment by altanborali16** [comment]
For sign up process we think and checked from papers such as [Everything about designing a great sign up experience](https://medium.com/@heyaliraza/everything-about-designing-a-great-sign-up-experience-66d91e31c220) it is important that Ask for most important information at sign-up and you can ask the others next, which means that email, username and password are enough for signup page and rest of the information can be filled from profile page. Moreover for the recommendation section for new comers we can show most popular communities as suggestions, after they are joining communities we can change the recommendation system, is that fine for you ?
